Makefile

```diff
@@ -15,14 +15,15 @@ endif
 MOVIE_SIZE=1280x720 # 640x360
 MOVIE_NAME=ivcap-cli.mp4
 
+LD_FLAGS="-X main.version=${GIT_TAG} -X main.commit=${GIT_COMMIT} -X main.date=${BUILD_DATE}"
 
 build: check
 	@echo "Building IVCAP-CLI..."
 	${GOPRIVATE_ENV_CMD} go mod tidy
-	go build -ldflags "-X main.version=${GIT_TAG} -X main.commit=${GIT_COMMIT} -X main.date=${BUILD_DATE}" -o ivcap${EXTENSION}
+	go -C ivcap build -ldflags ${LD_FLAGS}
 
-install: addlicense build
-	$(shell go install)
+install: addlicense check
+	go -C ivcap install -ldflags ${LD_FLAGS}
```

Comment:
Does that work? don't think there is `ivcap` dir.

Reply:
Yes, it works. This patch also moves cli.go to ivcap/cli.go, which is required by the go tooling.

Reply:
Ahh, yeah, there is a folder change.  Sorry ignored that. 

I think its better to be put under cmd/ivcap .